app/models/edition.rb

```diff
@@ -74,6 +76,12 @@ class Edition < ApplicationRecord
 
   delegate :content_id, :locale, to: :document
 
+  def auth_bypass_ids_are_uuids
+    unless auth_bypass_ids.all? { |id| UuidValidator.valid?(id) }
```

Comment:
Oh interesting - I didn't realise these were validated to be uuids. I've had to amend https://trello.com/c/jKRvjyAe/130-store-authbypassid-as-random-data-in-the-collection-publisher-database to reflect this.

Reply:
Strictly UUID or no entry! ⚔️